src/docs/usr-manual/solvers-ams.rst

```diff
@@ -68,9 +68,9 @@ In addition to the above quantities, AMS can utilize the following (optional)
 information:
 
 * The Poisson matrices :math:`A_\alpha` and :math:`A_\beta`, corresponding to
```

Comment:
I suppose you can remove this change now from the PR

Reply:
Yes, done at [79d5d84](https://github.com/hypre-space/hypre/pull/859/commits/79d5d84f3a91b65b9acb9888c15476d5b542b874)